extism_pdk.go

```diff
@@ -230,19 +234,19 @@ func GetVarInt(key string) int {
 
 func SetVarInt(key string, value int) {
 	keyMem := AllocateBytes([]byte(key))
-	defer keyMem.Free()
+	defer mem.Free()
```

Comment:
Perhaps I'm missing its definition, but where does `mem` come from?

Reply:
Yikes, I should not have done that in a hurry. I'm on a flight now but will fix this later tonight. 
Sorry.